spec/v3/services/env_var/delete_spec.rb

```diff
@@ -28,6 +28,6 @@
     end
 
     example { expect(last_response.status).to eq 200 }
-    example { pending 'should we return an empty body here?' }
+    example { expect(JSON.parse(last_response.body)["id"]).to eq(env_var[:id]) }
```

Comment:
Agreed. I like just 200 with no body, but do we currently have any V3 responses that don't return a body? Do we want to make this a pattern for all of the DELETE posts? I think consistency is good. Would users or our web like a status message in the body? I don't think we send any status messages for any 200 (ok) responses, so I'm guessing the answer is no.